rita_exit/src/database/mod.rs

```diff
@@ -92,15 +93,6 @@ pub fn get_exit_info() -> ExitDetails {
     }
 }
 
-// lossy conversion, but it won't matter until 2.9 * 10^8 millenia from now
-pub fn secs_since_unix_epoch() -> i64 {
-    let start = SystemTime::now();
-    let since_the_epoch = start
-        .duration_since(UNIX_EPOCH)
-        .expect("Time went backwards");
-    since_the_epoch.as_secs() as i64
-}
-
```

Comment:
moved to rita_common

Reply:
this function should probably be replaced with usage of SystemTime now since it's more standard. Didn't know about it when I made this func.

Reply:
According to the docs on SystemTime, it looks like your function uses the standard way to get unix time from it, so I think it's a convenient utility fn. I actually added a second fn for converting from a SystemTime to secs using the same approach.